src/services/redeem_and_unstake/AcceptRedeemRequestService.ts

```diff
@@ -0,0 +1,337 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
add this service to services.ts & repositories.ts

Reply:
Done!